test/functional/test_framework/test_framework.py

```diff
@@ -1119,9 +1119,7 @@ def set_dash_test_params(self, num_nodes, masterodes_count, extra_args=None, evo
             extra_args = [[]] * num_nodes
         assert_equal(len(extra_args), num_nodes)
         self.extra_args = [copy.deepcopy(a) for a in extra_args]
-        self.extra_args[0] += ["-sporkkey=cP4EKFyJsHT39LDqgdcB43Y3YXjNyjb5Fuas1GQSeAtjnZWmZEQK"]
-        for i in range(0, num_nodes):
-            self.extra_args[i].append("-dip3params=2:2")
+        self.extra_args_from_options += ["-sporkkey=cP4EKFyJsHT39LDqgdcB43Y3YXjNyjb5Fuas1GQSeAtjnZWmZEQK", "-dip3params=2:2"]
```

Comment:
6a161abbce4310fa9ffd1763da93342b49ecad27: `sporkkey` should be set on node0 only. Also, we probably should be using `dash.conf` for options that are meant to be persistent, pls check 32373548a4.

Reply:
Why exactly sporkkey can't be specified for all nodes? It's out of scope for all tests except "feature_sporks.py" to care about sporks functionally

Reply:
You _can_ do that. But you shouldn't :D